client/components/LearnerViewDirective.js

```diff
@@ -517,9 +534,29 @@ tie.directive('learnerView', [function() {
           loadQuestion(questionId, questionSet.getIntroductionParagraphs());
         };
 
+        var displayAutoSaveTextForSeconds = function(displaySeconds) {
```

Comment:
triggerAutosaveNotification

("Text" is a bit vague and feels like it can refer to the code. The arg name means you don't really need the "ForSeconds", I think.)